### PR TITLE
Django <= 1.8 compatibility and travisci with tox tests

### PR DESCRIPTION
**Pull request by bocribbz**
Fixes to run on Django 1.4 to 1.8.

Add tox.ini to run tests.

### CODE REVIEW

**Comment by kylegibson-rldatix** [comment]
Since tox.ini isn't on master, are you able to manually start a test run for this issue before it is merged in?

How do you feel about forking this project onto the PolicyStat org, that way it's easier for the rest of us to manage if needed in the future? Also, I feel we should try to avoid if possible having requirements that point to personal repos. What are your thoughts?

On the PolicyStat side, since you finished this update, have you tried updating requirements to point to this branch and tested that it works?

The code otherwise looks good.

**Comment by kylegibson-rldatix** [comment]
I think the answers to the above questions are important so I am going to move this issue back to implementation on Trello.

**Comment by bocribbz** [comment]
Migrated under PolicyStat org.

**Comment by winhamwr** [comment]
I have the same questions Kyle had:

> Since tox.ini isn't on master, are you able to manually start a test run for this issue before it is merged in?

Maybe post output of a successful tox run? Even better, maybe do a separate PR that just adds Travis support and cherry-pick that to this PR, too?

> On the PolicyStat side, since you finished this update, have you tried updating requirements to point to this branch and tested that it works?

**Comment by bocribbz** [comment]
Added `.travis.yml` file in this branch and ran the tests:
https://travis-ci.org/PolicyStat/django-json-field/builds/91902217

**Comment by bocribbz** [comment]
On PolicyStat side, the PR to update requirements with this version, is all green #2356.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  - TOX_ENV=py26-django16
+  - TOX_ENV=py27-django14
+  - TOX_ENV=py27-django15
+  - TOX_ENV=py27-django16
+  - TOX_ENV=py27-django17
+  - TOX_ENV=py27-django18
+  - TOX_ENV=py33-django15
+  - TOX_ENV=py33-django16
+  - TOX_ENV=py33-django17
+  - TOX_ENV=py33-django18
```
Comment (kylegibson-rldatix) [inline .travis.yml:16]: Can we support py34 too? Assuming it doesn't require any additional effort...

**Comment by kylegibson-rldatix** [comment]
Looks good :+1:

**Pull request merged**